src/components/OSCALMetadata.js

```diff
@@ -595,11 +604,33 @@ export function OSCALMetadataLocation(props) {
 }
 
 function OSCALMetadataFieldArea(props) {
-  const { title, children } = props;
+  const { title, children, urlFragment } = props;
+
+  const [expand, setExpand] = React.useState(false);
+
+  const handleExpand = () => {
+    setExpand(!expand);
+  };
+
+  const handleFragment = useCallback(() => {
+    // Grab fragment identifier following hash character if fragment exists in location
+    const controlFragment = urlFragment !== "" ? urlFragment : null;
+    // Expand metadata accordion section if control fragment matches title
+    if (controlFragment === title.toLowerCase()) {
+      setExpand(true);
+    }
+  }, [urlFragment, title]);
+
+  useEffect(() => {
+    handleFragment();
+  }, [handleFragment]);
```

Comment:
Same here actually. I am confused why we can't just do:


```suggestion
  useEffect(() => {
    // Grab fragment identifier following hash character if fragment exists in location
    const controlFragment = urlFragment !== "" ? urlFragment : null;
    // Expand metadata accordion section if control fragment matches title
    if (controlFragment === title.toLowerCase()) {
      setExpand(true);
    }
  }, [urlFragment, title]);
```